src/otx/algorithms/detection/configs/detection/mobilenetv2_ssd/semisl/data_pipeline.py

```diff
@@ -1,156 +1,7 @@
 """Data Pipeline of SSD model for Semi-Supervised Learning Detection Task."""
 
-# Copyright (C) 2022 Intel Corporation
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions
-# and limitations under the License.
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
 
-# pylint: disable=invalid-name
 
-# This is from otx/mpa/recipes/stages/_base_/data/pipelines/ubt.py
-# This could be needed sync with incr-learning's data pipeline
-__img_scale = (864, 864)
```

Comment:
Is it ok to refer "base_semisl_data_pipeline" although img_scale is different?

Reply:
Current detection's semisl data pipeline is not optimized for each model, so I think unified recipe is better since it helps management anyway.